bin/run.js

```diff
@@ -32,13 +32,24 @@ if (argv.version) {
 // initialize reporter by attempting to load lib/reporters/${reporter}
 // defaults to 'simple'
 let reporter;
+let reporterOpts = {};
 if (fs.existsSync(Path.join(__dirname, '../lib/reporters', `${argv.reporter}.js`))) {
   reporter = require(`../lib/reporters/${argv.reporter}.js`);
 } else {
   console.error(`Reporter ${argv.reporter} not found.`);
   process.exit(1);
 }
 
+if (argv.reporterKeys) {
+  if (argv.reporter !== 'json') {
+    console.error('`--reporter-keys` option applies only to the `json` reporter.');
+    process.exitCode = 1;
+    return;
```

Comment:
Ah, my response is a little ambiguous. Let me rephrase: would you like me to submit a separate pull request that corrects the convention prior to merging this pull request?